lib/rooms/prometheus_stats.ex

```diff
@@ -31,5 +42,6 @@ defmodule SignalTower.PrometheusHTTPHandler do
     {:ok, reply, :no_state}
   end
 
+  @spec terminate(any, any, any) :: :ok
```

Comment:
I guess if we use spec then we should do it everywhere (and then this should be a separate PR. Please remove this for now.

Reply:
I wonder where this line is coming from. I guess the elixir vscode plugin sneaked it in. Will remove the line.

Reply:
yes...?

Reply:
yes? what?

Reply:
Didn't you want to remove the line?

Reply:
Sure

Reply:
Fixed